packages/core/crates/core-hopr/src/p2p.rs

```diff
@@ -266,7 +266,7 @@ pub(crate) async fn p2p_loop(
                             error!("Ticket aggregation: Failed send reply to {}", peer);
                         }
                     },
-                    TicketAggregationProcessed::Receive(_peer, acked_ticket, request) => {
+                    TicketAggregationProcessed::Receive(_peer, _acked_ticket, request) => {
```

Comment:
Unnecessary, this is uncommented in Lukas' PR which will be merged soon and the warning was an intentional reminder that this functionality is not there yet.